coin/slip44.go

```diff
@@ -8,14 +8,16 @@ type Coin struct {
 	Decimals uint   `json:"decimals"`
 }
 
+// List of coin indexes
```

Comment:
no need comment

Reply:
Actually `go` recommends adding comments` https://blog.golang.org/godoc-documenting-go-code, plus linter was demanding adding it.

Reply:
many open-source `go` projects do comments https://github.com/cosmos/cosmos-sdk/blob/develop/server/init.go https://github.com/trezor/blockbook/blob/master/db/sync.go

Reply:
Projects should follow godoc standard godoc.org/github.com/trustwallet/blockatlas
That’s why the linter was asking for it.
Feel free to add it but I’m going to document the whole source anyways once the code structure is set in stone.

Reply:
Comments will be added in #50